tox.ini

```diff
@@ -1,18 +1,18 @@
 [tox]
-envlist = py{27,36,37}-{static, dynamic, upgrade}
+envlist = py{27,36,37}-{release, source}-{static, dynamic, upgrade}
```

Comment:
Nice! Naming it `release` makes it easier to understand

Reply:
Thanks you!